### PR TITLE
expose NfcB/F/V/IsoDep in NfcManager, and update readme

### PR DESCRIPTION
**Pull request by whitedogg13**

### CODE REVIEW

**Comment by whitedogg13** [comment]
Mostly for readme update, please feel free to feedback!

File: README.md [hunk 1]
```diff
@@ -52,18 +52,26 @@ AppRegistry.registerComponent('NfcManagerDev', () => App);
...
 
 All methods in `NfcManager` return a `Promise` object and are resolved to different types of data according to individual API.
 
+* `Ndef` is an utility module to encode and decode some well-known NDEF format.
 * `ByteParser` is an utility module to encode and decode byte[] arrays (used in Mifare Classic technology).
+* `NfcTech` contains predefined constants for specific NFC technologies, which include `NfcA`, `NfcB`, `NfcF`, `NfcV`, `IsoDep` and `MifareClassic`.
+    * These constants should be used with `requestTechnology` (Android Only) to obtain a NFC technology handle, and use it to perform technology specific operations.
 
-* `Ndef` is an utility module to encode and decode some well-known NDEF format.
+Detailed API document is grouped into 6 parts:
```
Comment (poison) [inline README.md]: Unsure what you mean with this phrase. Do you mean "The API documentation is grouped into 6 parts"?

File: README.md [hunk 2]
```diff
@@ -276,34 +283,56 @@ _onTagDiscovered = tag => {
...
+### makeReadOnly() [Android only]
+Make the tag become read-only.
+
+> This method returns a promise:
+> * if resolved, the operation success and tag should become read-only.
+> * if rejected, it means either the request is cancelled, the operation fail or the operation is not supported in current tech handle.
+
+## NfcTech.NfcA / NfcB / NfcF / NfcV / IsoDep API [Android only]
+
+To use the these API, you first need to request the `NfcTech.Ndef` technology (see `requestTechnology`). Once you have the tech request, you can use the following methods:
```
Comment (poison) [inline README.md]: NfcTech.Ndef => NfcTech.NfcA/NfcTech.NfcB/NfcTech.NfcV/NfcTech.IsoDep

File: README.md (hunk 1 above)
Comment (whitedogg13) [reply README.md]: Good catch, thanks!

**Pull request merged**